gamestonk_terminal/common/behavioural_analysis/finbrain_model.py

```diff
@@ -0,0 +1,38 @@
+"""FinBrain Model"""
+__docformat__ = "numpy"
+
+import pandas as pd
+import requests
+
+
+def get_sentiment(ticker: str) -> pd.DataFrame:
+    """Gets Sentiment analysis provided by FinBrain's API
+
+    Parameters
+    ----------
+    ticker : str
+        Ticker to get the sentiment analysis from
+
+    Returns
+    -------
+    DataFrame()
+        Empty if there was an issue with data retrieval
+    """
+    result = requests.get(f"https://api.finbrain.tech/v0/sentiments/{ticker}")
+    sentiment = pd.DataFrame()
+    if result.status_code == 200:
+        if "sentimentAnalysis" in result.json():
+            sentiment = pd.DataFrame.from_dict(
+                result.json()["sentimentAnalysis"], orient="index"
+            )
+            sentiment.index = pd.to_datetime(sentiment.index).to_pydatetime()
+            sentiment.index.name = "date"
+            sentiment.columns = ["Sentiment Analysis"]
+        else:
+            print("Unexpected data format from FinBrain API")
```

Comment:
So I was thinking about this here along with the reddit ones.  Should we raise an error or something?  Im envisioning a user using this in a Jupyter nb and getting an empty dataframe with no rationale.  If the print is in there it at least tells them what the error is?  Just an empty df doesn't tell us anything about what is wrong.

Reply:
True, but the idea is for us to avoid having prints in the model. This is because the "view" is what deals with the interface to the user. Also, if there's an issue in the code it will still break and give that error. In case the dataframe is empty, we know something went wrong.